test/functional/forknotify.py

```diff
@@ -41,11 +42,17 @@ def run_test(self):
         self.nodes[1].generate(1)
         self.sync_all()
 
-        with open(self.alert_filename, 'r', encoding='utf8') as f:
-            alert_text = f.read()
+        # Give bitcoind 10 seconds to write the alert notification
+        timeout = 10
```

Comment:
Depends. Travis catches a lot of timing edge cases. I don't think there's any problem making the timeout something high like 10 seconds since the while loop will break as soon as the file is written.